SignalMessaging/utils/UIUtil.m

```diff
@@ -23,42 +24,26 @@ + (void)removeRoundedBorderToImageView:(UIImageView *__strong *)imageView
     [[*imageView layer] setCornerRadius:0];
 }
 
-+ (completionBlock)modalCompletionBlock
-{
-    completionBlock block = ^void() {
-        [CurrentAppContext() setStatusBarStyle:UIStatusBarStyleLightContent];
-    };
-
-    return block;
-}
-
-+ (void)applyDefaultSystemAppearence
++ (void)setupSignalAppearence
 {
+    UINavigationBar.appearance.barTintColor = UIColor.ows_navbarBackgroundColor;
+    UINavigationBar.appearance.tintColor = UIColor.ows_navbarIconColor;
+    UIToolbar.appearance.barTintColor = UIColor.ows_navbarBackgroundColor;
+    UIToolbar.appearance.tintColor = UIColor.ows_navbarIconColor;
+
+    UIBarButtonItem.appearance.tintColor = UIColor.ows_navbarIconColor;
```

Comment:
So in terms of not needing the default system appearance, we need to test:

* Camera roll / media picker.
* Email composer.
* SMS Message composer.
* Contact editor.
* Document Picker.
* Sharing UI.

Right?

Reply:
Yup, I'll add this to the test plan.